test/components/structures/SpaceHierarchy-test.tsx

```diff
@@ -67,4 +79,137 @@ describe("SpaceHierarchy", () => {
             });
         });
     });
+
+    describe("toLocalRoom", () => {
+        let client: MatrixClient;
+        let roomV1: Room;
+        let roomV2: Room;
+        let roomV3: Room;
+
+        beforeEach(() => {
```

Comment:
If we are not mutating the mock created in the `beforeEach`, we can created them in the `describe` scope.